internal/channelz/funcs.go

```diff
@@ -298,13 +296,13 @@ func AddTraceEvent(id int64, depth int, desc *TraceEventDesc) {
 	for d := desc; d != nil; d = d.Parent {
 		switch d.Severity {
 		case CtUNKNOWN:
-			grpclog.InfoDepth(depth+1, d.Desc)
+			logger.InfoDepth(depth+1, d.Desc)
```

Comment:
Those are still logging to the wrong component.

Just change this function to take the extra `logger`. You won't need the unexported function `addTraceEventLogger` in logging.go.

Reply:
Removed addTraceEventLogger and updated AddTraceEvent to accept a logger. Also updated instances outside of internal/channelz where AddTraceEvent is used to use it with a logger.